### ISSUE TITLE
Uncaught TypeError: Cannot read properties of undefined (reading 'map')

### ISSUE THREAD

**Issue by MarketingPip** [spencermountain/compromise#1056]
There is an error with using ```.out()```. 

Example of usage that throws error:
```js
import nlp from "compromise"


let doc = nlp("some terms here")

console.log(doc.chunks().out("terms"))//
```

Line with error has been commented. 
```js
  // some handy ad-hoc outputs
  if (method === 'terms') {
    let list = []
    this.docs.forEach(s => {
      let terms = s.terms.map(t => t.text) // ERROR COME'S FROM HERE
      terms = terms.filter(t => t)
      list = list.concat(terms)
    })
    return list
  }
```

Adding this issue so I don't forget to resolve this (unless you get to it before me).

**Comment by spencermountain** [comment]
Good catch! Thank you

**Comment by MarketingPip** [comment]
@spencermountain - you're welcome! ps; check the gist I tagged you in!

**Comment by spencermountain** [comment]
fixed in `14.10.1` - thanks!
